base/services/tftpd/tftpd.cpp

```diff
@@ -1009,7 +1009,7 @@ void processRequest(void *lpParam)
                 continue;
 
             errno = 0;
-            req.bytesReady = (MYDWORD)outPtr - (MYDWORD)&req.mesout;
+            req.bytesReady = (const char*)outPtr - (const char*)&req.mesout;
```

Comment:
Maybe put that under a `#ifdef __REACTOS__` ? (since technically the TFTPD is 3rd-party).

Reply:
It has already been modified before me. MYDWORD is not original code.

Reply:
Thanks I didn't know that.